bin/zkServer-initialize.sh

```diff
@@ -113,6 +113,8 @@ initialize() {
     else
         echo "No myid provided, be sure to specify it in $ZOO_DATADIR/myid if using non-standalone"
     fi
+
+    date > "$ZOO_DATADIR/initialize"
```

Comment:
True enough, `touch` is sufficient. Using `date` is an optimization I've included in other scripts in the past as a way of sneaking a bit more information into an otherwise meaningless file but in this context it's probably just confusing.